### ISSUE TITLE
Allows Case Sensitive Search Only

### ISSUE THREAD

**Issue by talhaobject90** [netguru/ember-select-guru#8]
The select is working if the word is typed in the correct case . Even in the example page , to get John .. we should type  'John', simply 'john' wont work . Any way to over ride this?

![case2](https://cloud.githubusercontent.com/assets/7547924/11605879/4be5db44-9b31-11e5-9213-07e6bfb4068d.png)

![case](https://cloud.githubusercontent.com/assets/7547924/11605882/54c8b1e6-9b31-11e5-98f4-b6118133012d.png)

**Comment by jniechcial** [comment]
Thanks for posting an issue @talhaobject90! You are right that default behavior in select2-like implementation is case-insensitive. I just published package (1.1.1) that make use of case-insensitive search by default. However, if you would like to use case-sensitive now, you must implement custom search mechanism (we don't want to extend the API for case sensitive or insensitive option).

Fixed in #10
